src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterMetricsHttpServer.cs

```diff
@@ -112,27 +113,91 @@ private void WorkerThread()
                     ctx.Response.StatusCode = 200;
                     ctx.Response.ContentType = PrometheusMetricBuilder.ContentType;
 
-                    using var output = ctx.Response.OutputStream;
-                    using var writer = new StreamWriter(output);
-                    foreach (var metric in this.exporter.Metrics)
+                    using (var output = ctx.Response.OutputStream)
```

Comment:
why are you moving back from C#8 syntax?

Reply:
Moved back to new syntax. (changed by mistake while handling a merge conflict)